api/src/main/java/edu/cornell/mannlib/vitro/webapp/search/controller/PagedSearchController.java

```diff
@@ -124,7 +126,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
                 response.setCharacterEncoding("UTF-8");
                 response.setContentType("text/csv;charset=UTF-8");
-                response.setHeader("Content-Disposition", "attachment; filename=search.csv");
+                String date = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd-HHmmss"));
+                String applicationName = vreq.getAppBean().getApplicationName();
+                String fileName = date + "_" + applicationName + "_searchterm.csv";
```

Comment:
Should we replace _searchterm.csv with _searchresults.csv?

I suppose it is risky to include search term in the name of the file due to some potential irregular file name

Reply:
Yes, maybe that is a good solution, I did not think of strange search queries that could break file name conventions.